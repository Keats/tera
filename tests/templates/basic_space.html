<h1>{{object.doesnt_exist}}</h1>