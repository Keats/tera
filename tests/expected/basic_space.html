<h1 
	style="font-weight: bold" onclick="javascript: do_something()"
>Hello</h1 >